build/configs/sidk_s5jt200/README.md

```diff
@@ -8,6 +8,7 @@ Samsung IoT Development Kit for S5JT200 chipset.
 > [Environment Set-up](#environment-set-up)  
 > [How to program a binary](#how-to-program-a-binary)  
 > [ROMFS](#romfs)  
+> [Using GDB](#gdb)
```

Comment:
One more thing, you should add two spaces at end of line. It makes this line has different line with below.
Currently, it shows,
```
Using GDB Configuration Sets
```
Two spaces shows,
```
Using GDB
Configuration Sets
```